include/mbedtls/cipher.h

```diff
@@ -183,33 +190,40 @@ typedef struct mbedtls_cipher_base_t mbedtls_cipher_base_t;
 typedef struct mbedtls_cmac_context_t mbedtls_cmac_context_t;
 
 /**
- * Cipher information. Allows cipher functions to be called in a generic way.
+ * Cipher information. Allows calling cipher functions 
+ * in a generic way.
  */
 typedef struct {
-    /** Full cipher identifier (e.g. MBEDTLS_CIPHER_AES_256_CBC) */
+    /** Full cipher identifier. For example, 
+	 * MBEDTLS_CIPHER_AES_256_CBC.
```

Comment:
Blocker: this line starts with a tab. One of our sanity check scripts (`tests/scripts/check-names.sh`) expects that comment continuation lines start with a space. It doesn't recognize tabs and that causes it to complain about this line.

Here and more generally, please use spaces only, not tabs.

In any case, `MBEDTLS_CIPHER_AES_256_CBC` should be formatted as code.